### ISSUE TITLE
ECMAScript2015 sample code aesthetic bug (class="list-group-item")

### ISSUE THREAD

**Issue by MSoup** [jchadwick/EssentialTypeScriptBook#14]
There are 5 instances where class="list-group-item" has an extra "}", which TypeScript / Linting does not detect. The result is that bootstrap does not pick up on the list-group-item class and does not style the to-do list item correctly.

An example code snippet from within the book is

```js
	var todo = {
	    id: 123,
	    name: "Pick up drycleaning",
	    completed: true
	}
	
	container.innerHTML = `
	    <div todo='${todo.id}' class="list-group-item}">
	        <i class="[[ If Todo is complete, then "hidden" ]] text-success glyphicon glyphicon-ok"></i>
	        <span class="name">${todo.name}</span>
	    </div>
	`
```
